.circleci/config.yml

```diff
@@ -0,0 +1,66 @@
+version: 2.1
+executors:
+  ruby:
+    parameters:
+      version:
+        description: "Ruby version number"
+        default: "2.7.2"
```

Comment:
Bummer this ruby version is dup'd so many times.

Reply:
... twice in this file, `.ruby-version`... and maybe elsewhere?

Reply:
Unfortunately I had to add them in order to enable matrix-based testing (i.e. to run the rspec job against multiple ruby versions). I think the benefit of the test coverage outweighs the repetitiveness in this case.

Generally it seems like this is a common problem with CircleCI config where each job has to explicitly specify a version tag, like this example in node projects: https://github.com/carbonfive/spraygun-express/blob/47532ce00ebd055561e65e99a087ea4a30caa27a/.circleci/config.yml#L6-L27

Also on Rails projects it is common for the ruby version to be repeated in at least three places:

- Gemfile
- .ruby-version
- .circleci/config.yml